### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by scrafy**

### CODE REVIEW

File: staff/joseluis-juste/app-template/about.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    <aside>
+        <ul>
+            <li><a href="about.html">About</a></li>
+            <li><a href="contact.html">Contact</a></li>
+            <li><a href="login.html">Login</a></li>
+            <li><a href="news.html">News</a></li>
+            <li><a href="profile.html">Profile</a></li>
+            <li><a href="home.html">Home</a></li>
+            <li><a href="register.html">Register</a></li>
+        </ul>
```
Comment (aleixcam) [inline staff/joseluis-juste/app-template/about.html]: why there's a duplicated menu? btw in the first you left the ul/li structure and in the second you left the nav tag

File: staff/joseluis-juste/app-template/contact.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    <aside>
+        <ul>
+            <li><a href="about.html">About</a></li>
+            <li><a href="contact.html">Contact</a></li>
+            <li><a href="login.html">Login</a></li>
+            <li><a href="news.html">News</a></li>
+            <li><a href="profile.html">Profile</a></li>
+            <li><a href="home.html">Home</a></li>
+            <li><a href="register.html">Register</a></li>
+        </ul>
```
Comment (aleixcam) [inline staff/joseluis-juste/app-template/contact.html]: same as above

File: staff/joseluis-juste/app-template/about.html (hunk 1 above)
Comment (scrafy) [reply staff/joseluis-juste/app-template/about.html]: I know and in a real app i wouldn´t do it. I put it there only for fill with some of information

File: staff/joseluis-juste/app-template/contact.html (hunk 2 above)
Comment (scrafy) [reply staff/joseluis-juste/app-template/contact.html]: I know and in a real app i wouldn´t do it. I put it there only for fill with some of information

File: staff/joseluis-juste/app-template/about.html (hunk 1 above)
Comment (manuelbarzi) [reply staff/joseluis-juste/app-template/about.html]: remove it, as it breaks good practices. semantics should be clear, but content too.

File: staff/joseluis-juste/app-template/contact.html (hunk 2 above)
Comment (manuelbarzi) [reply staff/joseluis-juste/app-template/contact.html]: remove it, or change for any other content (lorem ipsum?), but not confuse the reviewers with content you only suppose that works to be there just for fulfilling space.

File: staff/joseluis-juste/app-template/home.html [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+        <a href="contact.html">Contact</a>
+        <a href="login.html">Login</a>
+        <a href="news.html">News</a>
+        <a href="profile.html">Profile</a>
+        <a href="home.html">Home</a>
+        <a href="register.html">Register</a>
+    </nav>
+
+    <aside>
+        <ul>
```
Comment (manuelbarzi) [inline staff/joseluis-juste/app-template/home.html]: remove repeated navigation links from here. html content should be right and not confusing.

File: staff/joseluis-juste/app-template/about.html (hunk 1 above)
Comment (scrafy) [reply staff/joseluis-juste/app-template/about.html]: Ok, It has been fixed

File: staff/joseluis-juste/app-template/contact.html (hunk 2 above)
Comment (scrafy) [reply staff/joseluis-juste/app-template/contact.html]: Ok, It has been fixed

File: staff/joseluis-juste/app-template/home.html (hunk 3 above)
Comment (scrafy) [reply staff/joseluis-juste/app-template/home.html]: Ok, It has been fixed

**Pull request merged**